modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -159,7 +159,9 @@ export class MessengerService {
         debugMessages('could not find a bot for page id =', pageId)
         continue
       }
-
+      if (!messages) {
+        return
+      }
```

Comment:
Just a quick question; the previous "if" lets the for loop continue if the value is not correct, would the same thing apply for the messages prop? I don't know the api of messenger so I don't know what are the possible values of "body.entry"

Reply:
@allardy not sure whats your question but body.entry in messenger could be policy-enforcement messages, thread-control messages,  "standby" messages as FB supports multiple simultaneous apps receiving the same message (but only 1 active).

I use standby and thread-control to switch between apps, HITL and to keep context+copy of the conversation before/after the bot kicked in. 

Did that somehow shed some light? This fix is just a way to be able to subscribe to all those event types (standby/thread etc) without errors.